src/GitHub.InlineReviews/Tags/InlineCommentTagger.cs

```diff
@@ -187,16 +185,21 @@ void Initialize()
 
             if (session == null)
             {
-                managerSubscription = sessionManager.WhenAnyValue(x => x.CurrentSession).Subscribe(SessionChanged);
+                managerSubscription = sessionManager.WhenAnyValue(x => x.CurrentSession).Subscribe(x => ForgetWithLogging(SessionChanged(x)));
             }
             else
             {
-                SessionChanged(session);
+                ForgetWithLogging(SessionChanged(session));
             }
 
             initialized = true;
         }
 
+        static void ForgetWithLogging(Task task)
```

Comment:
Ah yeah, that problem keeps coming up. We need to make logging the lowest level of our architecture so we can log from anywhere. We should handle this in #573 when we get time to finish that PR off.

For the moment, this seems like a workable hack. Should maybe add a note to #573 explaining that we have this hack here?